tests/suite/test_suite.py

```diff
@@ -4,20 +4,26 @@
 import e2e_logging
 import reconfiguration
 import recovery
+import rekey
 
 from inspect import signature, Parameter
 
-# For now, these are hardcoded.
+# TODO: For now, these are hardcoded. Indeed, late join after recovery is not yet supported.
+# https://github.com/microsoft/CCF/issues/315
 tests = [
```

Comment:
Leaving this as is for now to get some proofs that the two tests can run in an interleaved fashion. I'll revert before merging.